package.json

```diff
@@ -34,10 +34,11 @@
     "url": "https://github.com/edx/frontend-template-application/issues"
   },
   "dependencies": {
+    "@edx/brand": "npm:@edx/brand-openedx@^1.1.0",
     "@edx/frontend-component-footer": "10.0.11",
     "@edx/frontend-component-header": "2.0.6",
     "@edx/frontend-platform": "1.3.7",
-    "@edx/paragon": "12.0.5",
+    "@edx/paragon": "^12.0.5",
```

Comment:
nit: Just to follow the existing version convention in the package.json, the other dependencies are pinned to a specific version rather than using semver formatting (e.g., `12.0.5` vs. `^12.0.5`). Same with the semver formatting in the `@edx/brand` line above, too.